topydo/ui/TodoListWidget.py

```diff
@@ -240,6 +261,8 @@ def resolve_action(self, p_action_str, p_size=None):
                 self._execute_on_selected(cmd, execute_signal)
             else:
                 urwid.emit_signal(self, execute_signal, cmd)
+
+            urwid.emit_signal(self, 'save_cmd', cmd, execute_signal)
         else:
```

Comment:
I think this signal is redundant. The main class should save the executed command transparently. Let a column just emit 'repeat last command' and let it not worry whether it was saved.